components/reset_filter_button.py

```diff
@@ -7,7 +7,7 @@ def get_component():
         button = dbc.Button(
             "Återställ",
             id="reset_filter_button",
-            color="danger",
-            style={"margin-left": "50em", "height": "40px"},
+            color="link",
+            style={"height": "2.5em"},
```

Comment:
I think there's a bootstrap class for making the button size larger.
Maybe try `className="btn-lg"`?